rclcpp/test/test_duration.cpp

```diff
@@ -78,22 +124,24 @@ TEST(TestDuration, chrono_overloads) {
 }
 
 TEST(TestDuration, overflows) {
-  rclcpp::Duration max(std::numeric_limits<rcl_duration_value_t>::max());
-  rclcpp::Duration min(std::numeric_limits<rcl_duration_value_t>::min());
+  rclcpp::Duration max_(std::numeric_limits<rcl_duration_value_t>::max());
```

Comment:
what's the purpose of renaming this variable?

Reply:
otherwise the linter will suggest to inculde <algorithm> since it assumes that min max are function calls

Reply:
that seems odd to me since this code was running already on our buildfarm.

Reply:
yeah because someone addes algothims header to quiet the linter ;)